src/components/utils/Markdown.tsx

```diff
@@ -0,0 +1,165 @@
+import { useState, useEffect, useMemo } from "react";
+import ReactMarkdown from "react-markdown";
+import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
+import { coy } from "react-syntax-highlighter/dist/esm/styles/prism";
+import remarkGfm from "remark-gfm";
+import styled from "@emotion/styled";
+import { Button, Box } from "@chakra-ui/react";
+import { CopyIcon } from "@chakra-ui/icons";
+import { Row } from "../../utils/chakra";
+import { copySnippetToClipboard } from "../../utils/clipboard";
+
+// Required to display accurate font sizes for markdown elements.
+const StyledMarkdownWrapper = styled(Box)`
```

Comment:
can we just assign the markdown tag an id or something and do this in index.css instead of using emotion?